RealmSwift-swift2.0/Tests/PerformanceTests.swift

```diff
@@ -85,7 +85,16 @@ class SwiftPerformanceTests: TestCase {
     }
 
     private func copyRealmToTestPath(realm: Realm) -> Realm {
-        try! NSFileManager.defaultManager().removeItemAtPath(testRealmPath())
+        do {
+            try NSFileManager.defaultManager().removeItemAtPath(testRealmPath())
+        }
+        catch let error as NSError {
+            XCTAssertTrue(error.domain == NSCocoaErrorDomain && error.code == 4)
```

Comment:
I don't know, and I didn't feel it was worth spending the time figuring it out until the bigger problem of things not working at all is solved.